src/node-file-trace.ts

```diff
@@ -152,8 +158,10 @@ export class Job {
   async readlink (path: string) {
     const cached = this.symlinkCache.get(path);
     if (cached !== undefined) return cached;
+    await this.fileIOQueue.acquire();
```

Comment:
Any reason why this acquire is outside the `try` but the one on line 199 is inside the `try`?

Reply:
It's okay to `acquire` out of `try` block, I'll move the line `199` out of `try`